code/src/App.js

```diff
@@ -1,9 +1,29 @@
 import React from 'react'
+import { BrowserRouter, Redirect, Route, Switch } from 'react-router-dom'
+
+import {MovieList} from './components/MovieList'
+import {MovieDetails} from'./pages/MovieDetails'
+import {NotFound} from './pages/NotFound'
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+   
+    /* BrowserRouter, Switch and Route to decide which page to display */
+    <BrowserRouter>
+      <Switch>
+        <Route path="/"exact>
+          <MovieList />
+        </Route>
+        <Route path="/movies/:slug" exact>
+          <MovieDetails />
+        </Route>
+        <Route path="/404" >
+          <NotFound />
```

Comment:
This route does not seem to work, I did not manage to get to it, it just displayed netlifys 404 not found page. It did not work in my project either, so guess we'll have to look into why it works on our local machines, but not  when deployed.